plugins/inputs/kafka_consumer/README.md

```diff
@@ -53,14 +63,14 @@ to use them.
   ## When set this tag will be added to all metrics with the topic as the value.
   # topic_tag = ""
 
+  ## The list of Kafka message headers that should be pass as metric tags
+  ## works only for Kafka version 0.11+, on lower versions the message headers 
+  ## are not available
+  # msg_headers_to_tags = ['Country','City']
```

Comment:
We should show the default value, so unless we are going to always include country and city, this should be empty. If you want, you can include an example in the description or call out that it is a list of strings

```suggestion
  # msg_headers_to_tags = []
```